installation/gcp/dm/vmclarity-server.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Creates the virtual machine with environment variables and startup script."""
+
+import vmclarity_install_script
+
+COMPUTE_URL_BASE = 'https://www.googleapis.com/compute/v1/'
+
+IMAGE_SELFLINK = "projects/ubuntu-os-cloud/global/images/ubuntu-2204-jammy-v20230630"
```

Comment:
this image does not have cloud-init installed by default, so you might need to install it as part of the startup-script, or alternatively use the container optimized image

Reply:
Did you check it? I remember checking it and it did not have

Reply:
Yeah I checked, without installing anything you can run `sudo journalctl -u cloud-init -u cloud-final` and see cloud-init running.